### PR TITLE
Add CreatedPagesList (T7236) 

### PR DESCRIPTION
**Pull request by R4356th**
Adds the CreatedPagesList extension per T7236.

``mediawiki`` pull request: miraheze/mediawiki#1603.

### CODE REVIEW

File: ManageWikiExtensions.php [hunk 1]
```diff
@@ -1182,6 +1182,19 @@
...
+		'createdpageslist' => [
+			'name' => 'CreatedPagesList',
+			'linkPage' => 'https://www.mediawiki.org/wiki/Extension:CreatedPagesList',
+			'var' => 'wmgUseCreatedPagesList',
+			'conflicts' => false,
+			'requires' => [],
+			'section' => 'specialpages',
+			'install' => [
+				'sql' => [
+					'createdpageslist' => '$IP/extensions/CreatedPagesList/sql/patch-createdpageslist.sql'
```
Comment (Reception123) [inline ManageWikiExtensions.php:1194]: You'll need to use "" instead of '' for the SQL path.

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: ManageWikiExtensions.php (hunk 1 above)
Comment (R4356th) [reply ManageWikiExtensions.php]: Done.

**Review by RhinosF1 — CHANGES_REQUESTED** [review]
See concerns in -sre around lack of actor usage and why it can't just store page_id

**Comment by Universal-Omega** [comment]
Decided to just decline the extension for now, since I don't currently have time to do upstream patch, if someone wants to then it can be re-evaluated but until then it is declined.

**Pull request closed without merging**